src/halmos/utils.py

```diff
@@ -109,6 +109,14 @@ def f_sha3_name(bitsize: int) -> str:
     return f"f_sha3_{bitsize}"
 
 
+def f_inv_sha3_name(bitsize: int) -> str:
+    return f"f_inv_sha3_{bitsize}"
+
+
+# TODO: explore the impact of using a smaller bitsize for the range sort
+f_inv_sha3_size = Function("f_inv_sha3_size", BitVecSort256, BitVecSort256)
```

Comment:
BitVecSort32 should work for the input size, right?

Reply:
does the max size of byte arrays that can appear during execution not exceed 2^32 bits (2^29 bytes)?

Reply:
that's definitely not realistic in the current EVM execution context. 2^20 is already huge (1MB)